core/services/feeds/service.go

```diff
@@ -1210,7 +1210,7 @@ func (s *service) generateJob(ctx context.Context, spec string) (*job.Job, error
 	case job.FluxMonitor:
 		js, err = fluxmonitorv2.ValidatedFluxMonitorSpec(s.jobCfg, spec)
 	case job.Workflow:
-		js, err = workflows.ValidatedWorkflowJobSpec(ctx, spec)
+		js, err = workflows.ValidatedWorkflowJobSpec(ctx, s.lggr, spec)
```

Comment:
Why do we need a logger for validation? Typically we want to either log something, (x)or return an error to be logged - not both.

Reply:
I needed it to call into the WASM creation.

Reply:
I would rather change the WASM runner to not need it if we're not going to use it.

Reply:
https://github.com/smartcontractkit/chainlink/pull/14458/files#r1770570163